Ginger/GingerCoreNET/Database/ActDBValidation.cs

```diff
@@ -146,8 +146,7 @@ public string SQL
         [IsSerializedForLocalRepository]
         public ObservableList<ActInputValue> QueryParams = new ObservableList<ActInputValue>();
 
-        [IsSerializedForLocalRepository]
-        public ObservableList<CosmosPatchInputValues> CosmosPatchInputValues = new ObservableList<CosmosPatchInputValues>();
+        public ObservableList<ActInputValue> CosmosPatchInputValues = new ObservableList<ActInputValue>();
```

Comment:
Please rename the field to generic name UpdateOperationInputValues or something. 
Because ActDBValidation is generic class and same field might be used for other DB types in future